nyc_taxis/challenges/common/autoscale-v2-schedule.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "delete-index",
+  "operation": {
+    "operation-type": "delete-index"
+  }
+},
+{
+  "name": "create-index",
+  "operation": {
+    "operation-type": "create-index",
+    "settings": {%- if index_settings is defined %} {{index_settings | tojson}} {%- else %} {
+      {# non-serverless-index-settings-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%}
+      "index.store.type": "{{store_type | default('fs')}}",
+      {%- endif -%}{# non-serverless-index-settings-marker-end #}
+      "index.codec": "best_compression"
+    }{%- endif %}
+  }
+},
+{
+  "name": "check-cluster-health",
+  "operation": {
+    "operation-type": "cluster-health",
+    "index": "nyc_taxis",
+    "request-params": {
+      "wait_for_status": "{{cluster_health | default('green')}}"
+    },
+    "retry-until-success": true
+  }
+}
+{%- set p_as_clients = (as_clients | default([]))%}
```

Comment:
What do you think about including some default values for these parameters? Something basic like `[8, 16, 32]` for clients and whatever makes sense for the other lists would be fine.

Reply:
Sure, addressed in https://github.com/elastic/rally-tracks/pull/569/commits/3c3d709f619a1b85556f82634134bcb8f2ca0205.